azerothcore/azerothcore-wotlk
# Core/Commands [Implement kickdelay command]

## Changes Proposed:
-  Implemented kickdelay/bandelay commands

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2817

## Tests Performed:

- Builds and test ingame


## How to Test the Changes:


Example commands: 
.kickdelay test Cheating 10 <- 10 means seconds
.bandelay ip 147.232.22.53 1d Cheating 10
.bandelay account test 1d Cheating 10

1. Type .kickdelay $charactername $reason $delay_time or .bandelay account $acc_name $bantime $reason $delay_time or .bandelay ip $acc_ip $bantime $reason $delay_time
2. Announce from $reason will appear at client.
3. Wait $delay_time given time until kick/ban.
4. Profit?

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
